datasets/pass/data.json

```diff
@@ -0,0 +1,171 @@
+      "sha256": "83e07eee33df5fcc63a4b0db4e94abb4" ,
+      "encodingFormat": "application/x-tar"
+    },
+    {
+      "name": "pass7",
+      "@type": "FileObject",
+      "contentUrl": "https://zenodo.org/record/6615455/files/PASS.8.tar",
+      "sha256": "3f708993aa168b57306e5de08ba71373" ,
+      "encodingFormat": "application/x-tar"
+    },
+    {
+      "name": "pass8",
+      "@type": "FileObject",
+      "contentUrl": "https://zenodo.org/record/6615455/files/PASS.9.tar",
+      "sha256": "f4f87af4327fd1a66dd7944b9f59cbcc" ,
+      "encodingFormat": "application/x-tar"
+    },
+    {
+      "name": "pass9",
+      "@type": "FileObject",
+      "contentUrl": "https://zenodo.org/record/6615455/files/PASS.1.tar",
+      "sha256": "0429e77aa029f9ff6e1a6a78efde1c33" ,
+      "encodingFormat": "application/x-tar"
+    },
+    {
+      "name": "image-files",
+      "@type": "FileSet",
+      "containedIn": [
+        "#{pass0}",
+        "#{pass1}",
+        "#{pass2}",
+        "#{pass3}",
+        "#{pass4}",
+        "#{pass5}",
+        "#{pass6}",
+        "#{pass7}",
+        "#{pass8}",
+        "#{pass9}"
+      ],
+      "includes": "*.jpg",
+      "encodingFormat": "image/x-jpg"
+    }
+  ],
+  "recordSet": [
+    {
+      "name": "images",
+      "@type": "ml:RecordSet",
+      "source": "#{image-files}",
+      "key": "#{hash}",
+      "field": [
+        {
+          "name": "hash",
+          "@type": "ml:Field",
+          "description": "The hash of the image, as computed from YFCC-100M.",
+          "dataType": "Text",
+          "source": {
+            "data": "#{image-files/filename}",
+            "applyTransform": {"regex": "([^\\/]*)\\.jpg"}
+          },
+          "references": "#{metadata/image_hash}"
+        },
+        {
+          "name": "image_content",
+          "@type": "ml:Field",
+          "description": "The content of the image.",
+          "dataType": "ImageObject",
+          "source": "#{image-files/content}"
+        },
+        {
+          "name": "creator_uname",
+          "@type": "ml:Field",
+          "description": "Unique name of photo creator.",
+          "dataType": "String",
+          "source": "#{metadata/unickname}"
+        },
+        {
+          "name": "gps_coordinates",
+          "@type": "ml:Field",
+          "description": "GPS coordinates where the image was taken.",
+          "dataType": "GeoCoordinates",
+          "subField": [
+            {
+              "name": "latitude",
+              "@type": "ml:Field",
+              "source": "#{metadata/latitude}"
+            },
+            {
+              "name": "longitude",
+              "@type": "ml:Field",
+              "source": "#{metadata/longitude}"
+            }
+          ]
+        },
+        {
+          "name": "date_taken",
+          "@type": "ml:Field",
+          "description": "The date the photo was taken.",
+          "dataType": "Date",
```

Comment:
Should we (have the capability to) specify the DateTime format of this String? And if so, how?
This is an example of the values in this file: `2004-05-17 00:44:29.0`

Reply:
Yes! I added a format field, which we should describe in the spec. We could nest it under applyTransform, but I kinda like keeping them separate.